### PR TITLE
Port Event model over to postgres

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

This ports over the Event model into Postgres, including a rake task to get things out of Mongo.

To verify:

* check out main and do a fresh `rails db:seed`, then log in and head to the CM dashboard. Nice lookin' events there down at the bottom. Take a screencap of the first few for reference
* Check out this branch and do `rails db:migrate` to move to the new branch. Reload the CM dashboard and confirm that the events disappear
* Run `rails migrate_to_pg:event` to port over data cleanly, then reload the CM dashboard and see your events show back up
* Log a call on a patient by clicking the phone icon and then Left voicemail. Doesn't matter which one. Reload the page and observe the new event showing up - this proves that making new events works.

This pull request makes the following changes:
* Port Event model to postgres/ActiveRecord
* write migration rake task

No view changes

It relates to the following issue #s: 
* Bumps #2072

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

**Comment by colinxfleming** [comment]
This is shovel ready and looking good! Going to queue this up for review after the config pr goes in.

File: db/migrate/20201212010730_create_events.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class CreateEvents < ActiveRecord::Migration[6.0]
+  def change
+    create_table :events do |t|
+      t.string :cm_name
+      t.integer :event_type
+      t.string :line
```
Comment (colinxfleming) [inline db/migrate/20201212010730_create_events.rb:6]: note to self: should lines continue to be strings or should we int them?

**Comment by colinxfleming** [comment]
Config is in, and this is next. Kicking it up for review. None of the code here has changed since I've written it, so this PR and its testing instructions should be good to go.

**Comment by lomky** [comment]
Testing looks good to me

Step | Screenshot
--|--
Main fresh seed reference | Freshly seeded even list on main
Empty event list after migration, before rake | Migration run on this PR showing empty list
Event list after the rake task, matches main | Event list on PR branch after the rake task, identical to before
New entry | Event list on PR branch showing a new entry

**Review by lomky — APPROVED** [review]

File: test/models/event_test.rb [hunk 2]
```diff
@@ -9,28 +9,34 @@ class EventTest < ActiveSupport::TestCase
...
-        @event.event_type = bad_status
-        refute @event.valid?
+      assert_raises ArgumentError do
+        @event.event_type = :notatype
+        @event.valid?
       end
 
-      valid_types = ['Reached patient', "Couldn't reach patient",
-                     'Left voicemail']
+      valid_types = [:reached_patient, :couldnt_reach_patient, :left_voicemail]
```
[the remark this answers is not in the document]
Comment (colinxfleming) [reply test/models/event_test.rb]: I think it's reasonable to do this here tbh! Bumped it

**Pull request merged**